### DISCUSSION TITLE
Add max stack override bool for possible base game containers and modders

### DISCUSSION THREAD

**Discussion by McFow1er** [FakeFishGames/Barotrauma#5133] [Ideas and suggestions]
The stacking system is great but a current limiting factor that I and a few others have encountered is the fact that currently `maxstack` takes the lowest value of the two, this makes setups where you can have a small max stack in your hotbar, then a container with a higher single item stack amount not currently possible. I propose an override bool tag over an automatic system to avoid potential problems with certain aspects of stacking. Along with this it would also be nice to have something similar to `containable` or using it even with a new tag like `target` for absolute item targeting to apply stacking tags and similar while not messing with anything's ability to be stored.

**Comment by PanmanS** [comment]
Actually a very cool idea, so i hope it will get implemented in future!
